packages/playwright-core/src/server/codegen/javascript.ts

```diff
@@ -106,7 +106,7 @@ export class JavaScriptLanguageGenerator implements LanguageGenerator {
       case 'navigate':
         return `await ${subject}.goto(${quote(action.url)});`;
       case 'select':
-        return `await ${subject}.${this._asLocator(action.selector)}.selectOption(${formatObject(action.options.length === 1 ? action.options[0] : action.options)});`;
+        return `await ${subject}.${this._asLocator(action.selector)}.selectOption(${formatObject(action.options.length > 1 ? action.options : action.options[0])});`;
```

Comment:
maybe just remove the test since to fix seems fine to land without a test?

Reply:
@mxschmitt Shall I just submit a PR without test then?

Reply:
That should work, fix seems non confrontational to land without a test.